src/main.cpp

```diff
@@ -2086,14 +2086,16 @@ static bool ActivateBestChainStep(CValidationState &state, CBlockIndex *pindexMo
 // or an activated best chain. pblock is either NULL or a pointer to a block
 // that is already loaded (to avoid loading it again from disk).
 bool ActivateBestChain(CValidationState &state, CBlock *pblock) {
-    CBlockIndex *pindexNewTip = NULL;
     CBlockIndex *pindexMostWork = NULL;
     do {
         boost::this_thread::interruption_point();
 
+        CBlockIndex *pindexNewTip;
```

Comment:
In the moved line, this variable was initialized to NULL, but no more. Maybe better to do so just in case so that potential bugs result in more predictable outcomes.